os_net_config/impl_ifcfg.py

```diff
@@ -1098,6 +1098,12 @@ def add_sriov_pf(self, sriov_pf):
         logger.info('adding sriov pf: %s' % sriov_pf.name)
         data = self._add_common(sriov_pf)
         logger.debug('sriov pf data: %s' % data)
+        utils.update_sriov_pf_map(sriov_pf.name, sriov_pf.numvfs,
```

Comment:
Sorry, in the recent revision the update_sriov_pf_map is triggered from objects.py @ https://github.com/os-net-config/os-net-config/blob/master/os_net_config/objects.py#L1637

Reply:
fixed